test/e2e/atlas/streams_test.go

```diff
@@ -0,0 +1,317 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || (atlas && generic)
```

Comment:
can you please create a dedicated target so we can assign these test to your team?

Reply:
Resolved in [e810158](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/e8101582d7e0faf61d3dbaac2117b6b75222fe99)